HW1/P4/P4.txt

```diff
@@ -1 +1,17 @@
 # Your discussion here
+My graph representation was a pair RDD where key is character
+and value is characters in the same book.
+
+Number of touched nodes
+'CAPTAIN AMERICA':
+6403 if all whitespaces in the character name  were stripped (which should be more accurate?)
+6408 if whitespaces were not stripped
```

Comment:
I like the extra consideration regarding whitespace. I didn't strip the whitespace, but I think stripping is the right way to go.